backend/infrahub/graphql/mutations/repository.py

```diff
@@ -228,15 +234,17 @@ async def mutate(
             branch=branch,
         )
 
-        message = messages.GitRepositoryImportObjects(
+        model = GitRepositoryImportObjects(
             repository_id=repository_id,
             repository_name=str(repo.name.value),
             repository_kind=repo.get_kind(),
             commit=str(repo.commit.value),
             infrahub_branch_name=branch.name,
         )
         if context.service:
-            await context.service.send(message=message)
+            await context.service.workflow.submit_workflow(
```

Comment:
I think it would be good to return the task id for this mutation moving forward.
i'll be useful to provide a better experience in the frontend as well at some point